src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.cs

```diff
@@ -75,7 +75,7 @@ public void TestFlowLog()
             NetworkResourcesController.NewInstance.RunPsTest("Test-FlowLog");
         }
 
-        [Fact]
+        [Fact(Skip = "There was a bug in API. After fix is deployed, we need to rerecord test.")]
```

Comment:
For these 2 skipped tests, can you provide a link to the bug? Then, someone can check the status on the bug to know when this test can be re-enabled.

Reply:
There is no link. I will take care of rerecording these tests by myself